NPEs with Picasso use

This has caused 51 crashes via Fabric.io.

    java.lang.RuntimeException: Transformation bitmapBorder(borderSize=2, cornerRadius=128, color=-1) crashed with exception.
       at com.squareup.picasso.BitmapHunter$3.run(BitmapHunter.java:434)
       at android.os.Handler.handleCallback(Handler.java:739)
       at android.os.Handler.dispatchMessage(Handler.java:95)
       at android.os.Looper.loop(Looper.java:135)
       at android.app.ActivityThread.main(ActivityThread.java:5274)
       at java.lang.reflect.Method.invoke(Method.java)
       at java.lang.reflect.Method.invoke(Method.java:372)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:909)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:704)
    Caused by: java.lang.NullPointerException: Attempt to read from field 'int android.graphics.Bitmap$Config.nativeInt' on a null object reference
       at android.graphics.Bitmap.createBitmap(Bitmap.java:817)
       at android.graphics.Bitmap.createBitmap(Bitmap.java:794)
       at android.graphics.Bitmap.createBitmap(Bitmap.java:761)
       at org.gdg.frisbee.android.view.BitmapBorderTransformation.transform(BitmapBorderTransformation.java:34)
       at com.squareup.picasso.BitmapHunter.applyCustomTransformations(BitmapHunter.java:429)
       at com.squareup.picasso.BitmapHunter.hunt(BitmapHunter.java:238)
       at com.squareup.picasso.BitmapHunter.run(BitmapHunter.java:159)
       at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:422)
       at java.util.concurrent.FutureTask.run(FutureTask.java:237)
       at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1112)
       at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:587)
       at java.lang.Thread.run(Thread.java:818)
       at com.squareup.picasso.Utils$PicassoThread.run(Utils.java:411)

--- a/app/src/main/java/org/gdg/frisbee/android/view/BitmapBorderTransformation.java
+++ b/app/src/main/java/org/gdg/frisbee/android/view/BitmapBorderTransformation.java
@@ -23,6 +23,10 @@ public class BitmapBorderTransformation implements Transformation {
 
     @Override
     public Bitmap transform(Bitmap source) {
+        if (source.getConfig() == null) {
+            return source;
+        }
+
         int width = source.getWidth();
         int height = source.getHeight();
 

